code/modules/mining/equipment/regenerative_core.dm

```diff
@@ -86,6 +86,18 @@
 			H.revive(full_heal = 1)
 			qdel(src)
 
+/obj/item/organ/regenerative_core/attack_self(mob/user) //Knouli's first hack! Allows for the use of the core in hand rather than needing to click on the target, yourself, to selfheal. Its a rip of the proc just above - but skips on distance check and only uses 'user' rather than 'target'
+	if(ishuman(user)) //Check if user is human, no need for distance check as it's self heal
+		var/mob/living/carbon/human/H = user //Set H to user rather than target
+		if(inert) //Inert cores are useless
```

Comment:
Make sure to check if they are dead or not other wise it can work on dead players.

Reply:
They literally left a comment in the code and in the PR why the check is gone, people dead can't use inhands anyways, can they?

Reply:
how is a dead player going to use that

Reply:
What if they're dead and it's in their hand, and they're still in their body and press z?

Reply:
This change only influences cores held in the hand, not implanted cores. Implanted cores will retain preexisting functionality - which is to activate automatically without player activation upon threshold of entering into crit, if I'm not mistaken. This pull utilizes a widespread proc, attack_self, which is used by near any item I was able to view in the code already - which ought to have handling for dead characters attempting to use items.

Which is all to say, upon death, you drop the core you held in hand.
![image](https://user-images.githubusercontent.com/25857923/69192952-b0eeeb80-0af3-11ea-99e9-09984b544a46.png)


In the case of the use of an anti-drop implant, the standing proc, attack_self has handling in-built already to prevent use of the core, even in death.
![image](https://user-images.githubusercontent.com/25857923/69193302-6f127500-0af4-11ea-83d6-df8704469123.png)


There presently stands no logical situation where a dead character will have a core in-hand, and able to use it. There exists code already to prevent the dead from using the core and a death check is unnecessary outside of redundancy.